frameworks/cis-v1.23-t1.0.1.json

```diff
@@ -5,6 +5,11 @@
         "version": "v1.0.1",
         "armoBuiltin": true
     },
+    "scanningScope": {
+        "matches": [
+            "cloud"
```

Comment:
does `cloud` scope mean his fw will only run on cloud managed clusters? This fw is suitable for others as well (minikube..)

Reply:
@rcohencyberarmor well then this scope needs to be changed because this framework is suitable also for clusters that are not cloud managed